src/Traits/HasPermissions.php

```diff
@@ -7,20 +7,47 @@
 trait HasPermissions
 {
     /**
-     * Grant the given permission to a role.
+     * Grant the given permission(s) to a role.
      *
-     * @param  $permission
+     * @param string|array|Permission|\Illuminate\Support\Collection $permissions
+     * @param array $params
      *
      * @return HasPermissions
      */
-    public function givePermissionTo($permission)
+    public function givePermissionTo($permissions, ...$params)
     {
-        $this->permissions()->save($this->getStoredPermission($permission));
+        $joinings = $this->resolveJoiningAttributes($params);
```

Comment:
I realize as i'm typing to explain this it's silly, I was trying to give the user the option to use the joinings option that saveMany on Eloquent has. I'll remove this because in this particular case we can work without it.